### PR TITLE
ign -> gz Header Migration : gz-gui

### PR DESCRIPTION
**Pull request by methylDragon**
See https://github.com/ignition-tooling/release-tools/issues/711

### CODE REVIEW

**Review by chapulina — APPROVED** [review]
LGTM as long as we don't get any unexpected test failures. This package has lots of flaky tests, I'll make another comment once all CI is back.

File: src/plugins/CMakeLists.txt [hunk 1]
```diff
@@ -17,27 +17,38 @@
...
+    set(options)
+    set(oneValueArgs)
+    set(multiValueArgs SOURCES QT_HEADERS PUBLIC_LINK_LIBS PRIVATE_LINK_LIBS)
+    cmake_parse_arguments(gz_gui_add_library "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
 
-  cmake_parse_arguments(ign_gui_add_library "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
+    set(gz_gui_add_library_skip_parsing true)
+    gz_gui_add_library(${library_name})
+endfunction()
+function(gz_gui_add_library library_name)
```
[the remark this answers is not in the document]
Comment (methylDragon) [reply src/plugins/CMakeLists.txt:19]: No longer tick-tocked in https://github.com/gazebosim/gz-gui/pull/395/commits/bff9d8661166d87d8c14bec7bb38f2c22d550072

[the remark this answers is not in the document]
Comment (chapulina) [reply src/plugins/CMakeLists.txt:19]: All the uses will need to be updated as well, CI is failing with

```
CMake Error at src/plugins/camera_tracking/CMakeLists.txt:1 (ign_gui_add_plugin):
  Unknown CMake command "ign_gui_add_plugin".
```

**Review by chapulina — APPROVED** [review]
CI looks good too, 🚢 it!

**Pull request merged**